grails-app/controllers/kuorum/participatoryBudget/ParticipatoryBudgetController.groovy

```diff
@@ -259,6 +259,11 @@ class ParticipatoryBudgetController extends CampaignController {
         } else {
             filter.sort = new FilterDistrictProposalRDTO.SortDistrictProposalRDTO(field: FilterDistrictProposalRDTO.SortableFieldRDTO.PRICE, direction: DirectionDTO.ASC)
         }
+        if (params.voter){
+            String voter = params.voter
```

Comment:
May be, you should use the data of the user logged instead of param. This is for security reasons.